Source/WebCore/svg/SVGImageElement.cpp

```diff
@@ -132,11 +136,11 @@ bool SVGImageElement::haveLoadedRequiredResources()
 
 void SVGImageElement::didAttachRenderers()
 {
-    if (auto* imageObj = downcast<RenderSVGImage>(renderer())) {
-        if (imageObj->imageResource().cachedImage())
-            return;
+    SVGGraphicsElement::didAttachRenderers();
```

Comment:
I guess I will trust you on this, but it is really much simpler if this change is a strict rename. Those can in fact be rubber stamped, more or less.

Reply:
I will remember this for the next patch, sorry.